README_GPDB_ru.md

```diff
@@ -0,0 +1,207 @@
+# 1. Поддерживаемые версии Greenplum
+
+На данный момент поддерживается только Greenplum 6.
+
+# 2. Компиляция pgBackRest
+
+- Установить зависимости
+
+для CentOS 7:
+```
+yum install gcc openssl-devel libxml2-devel bzip2-devel libzstd-devel lz4-devel libyaml-devel
+```
+
+для ALT Linux p10
+```
+apt-get update
+apt-get install git gcc openssl-devel libxml2-devel bzip2-devel libzstd-devel libyaml-devel zlib-devel 
+```
+
+- Добавить в переменную окружения PATH путь к pg_config Greenplum
+
+- Скачать репозиторий
+```
+git clone https://github.com/arenadata/pgbackrest -b 2.50-ci
+```
+
+- Перейти в каталог с исходным кодом
+```
+cd pgbackrest/src/
+```
+
+- Запустить скрипт конфигурирования
+
+Без параметров будут включены оптимизация (-O2) и векторизация вычисления контрольных сумм страниц.
+```
+./configure
+```
+
+Конфигурирование для отладки рекомендуется выполнять командой
+```
+CFLAGS="-O0 -g3" ./configure --disable-optimize --enable-test
+```
+
+- Собрать
+
+Команда для сборки с использованием всех доступных ядер и без отображения исполняемых команд
+```
+make -j`nproc` -s
+```
+
+В результате в текущем каталоге появится исполняемый файл с именем `pgbackrest`.
+
+- Установить
+
+```
+sudo make install
+```
+По умолчанию исполняемый файл `pgbackrest` будет размещен в каталог `/usr/local/bin`. Это единственное действие, которое выполняется на этапе установки.
+
+
+# 3. Настройка
+- Создать каталоги для размещения бэкапа и логов. Пусть для теста это будут `/tmp/backup` и `/tmp/backup/log`.
+```
+mkdir -p /tmp/backup/log
+```
+
+- Создать конфигурационный файл
```

Comment:
Please add a sample cmd for creation a standard config file at standard location. It is trivial, but I think I think better to mention it (as long as we add such steps as 'mkdir').